src/jit/compiler.h

```diff
@@ -3098,6 +3102,12 @@ class Compiler
                                 bool                  mustExpand);
     bool compSupportsHWIntrinsic(InstructionSet isa);
     bool isScalarISA(InstructionSet isa);
+    int ivalOfHWIntrinsic(NamedIntrinsic intrinsic);
```

Comment:
Most of these lookup functions should still be `static`. They don't depend on any compiler state.

Reply:
But Codegen needs to call them.

Reply:
Yes, and it would call them as `Compiler::ivalOfHWIntrinsic`, rather than `comp->ivalOfHWIntrinsic`

Reply:
Is it better than a member function? I think they’re both static calls.

Reply:
The member function passes in the hidden `this` parameter and implies that it relies on the state of `compiler`.